endpoint.go

```diff
@@ -130,6 +137,11 @@ func (e *InEndpoint) Read(buf []byte) (int, error) {
 // The passed context can be used to control the cancellation of the read. If
 // the context is cancelled, ReadContext will cancel the underlying transfers,
 // resulting in TransferCancelled error.
+// When a USB device receives a read request, it doesn't know the size of the
```

Comment:
This is missing the first sentence of the documentation added to Read, i.e. “It's recommended to use buffer sizes that are multiples of EndpointDesc.MaxPacketSize to avoid overflows.”

Reply:
Done.